Module 2 Exercises.ipynb

```diff
@@ -138,10 +529,222 @@
+              "      <td>UKN0</td>\n",
+              "      <td>2022</td>\n",
+              "      <td>NaN</td>\n",
+              "      <td>NaN</td>\n",
+              "      <td>NaN</td>\n",
+              "      <td>NaN</td>\n",
+              "      <td>NaN</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>26241</th>\n",
+              "      <td>ED5-8</td>\n",
+              "      <td>25-64</td>\n",
+              "      <td>UKN0</td>\n",
+              "      <td>2023</td>\n",
+              "      <td>NaN</td>\n",
+              "      <td>NaN</td>\n",
+              "      <td>NaN</td>\n",
+              "      <td>NaN</td>\n",
+              "      <td>NaN</td>\n",
+              "    </tr>\n",
+              "  </tbody>\n",
+              "</table>\n",
+              "<p>26242 rows × 9 columns</p>\n",
+              "</div>"
+            ],
+            "text/plain": [
+              "      isced11    age   geo  year  Value               unit direct na_item  \\\n",
+              "0         NaN    NaN    AT  1995    NaN  PPS_EU27_2020_HAB    BAL     B5N   \n",
+              "1         NaN    NaN    AT  1995    NaN  PPS_EU27_2020_HAB    BAL     B6N   \n",
+              "2         NaN    NaN    AT  1996    NaN  PPS_EU27_2020_HAB    BAL     B5N   \n",
+              "3         NaN    NaN    AT  1996    NaN  PPS_EU27_2020_HAB    BAL     B6N   \n",
+              "4         NaN    NaN    AT  1997    NaN  PPS_EU27_2020_HAB    BAL     B5N   \n",
+              "...       ...    ...   ...   ...    ...                ...    ...     ...   \n",
+              "26237   ED5-8  25-64  UKN0  2019   40.6                NaN    NaN     NaN   \n",
+              "26238   ED5-8  25-64  UKN0  2020    NaN                NaN    NaN     NaN   \n",
+              "26239   ED5-8  25-64  UKN0  2021    NaN                NaN    NaN     NaN   \n",
+              "26240   ED5-8  25-64  UKN0  2022    NaN                NaN    NaN     NaN   \n",
+              "26241   ED5-8  25-64  UKN0  2023    NaN                NaN    NaN     NaN   \n",
+              "\n",
+              "       value  \n",
+              "0        NaN  \n",
+              "1        NaN  \n",
+              "2        NaN  \n",
+              "3        NaN  \n",
+              "4        NaN  \n",
+              "...      ...  \n",
+              "26237    NaN  \n",
+              "26238    NaN  \n",
+              "26239    NaN  \n",
+              "26240    NaN  \n",
+              "26241    NaN  \n",
+              "\n",
+              "[26242 rows x 9 columns]"
+            ]
+          },
+          "execution_count": 142,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "df_education_tert_2564 = df_education_clean[df_education_clean['age'] == \"25-64\"] \n",
+        "df_education_tert_2564 = df_education_tert_2564[df_education_tert_2564['isced11'] == \"ED5-8\"]\n",
```

Comment:
A slightly less verbose way to subset the data is to use `pandas.DataFrame.query`: 
```
df_education_tert_2564 = df_education_tert_2564.query('isced11 == "ED5-8"')

```